cloudflared/rootfs/etc/cont-init.d/10-cloudflared-config.sh

```diff
@@ -113,7 +113,7 @@ createCertificate() {
     bashio::log.notice
     cloudflared tunnel login
 
-    bashio::log.green "Authentication successfull, moving auth file to config folder"
+    bashio::log.green "Authentication successfull, moving auth file to the '${data_path}' folder"
```

Comment:
Like the suggestion, would slightly change again:

`bashio::log.green "Authentication successful, moving auth file to the custom data folder: '${data_path}'"`

Reply:
This variable doesn't necessarily contain the custom data folder. If no data_folder option is provided it will simply contain /data. So I think the log is correct. But I would change method to bashio::log.**info**

Reply:
Well, this is totally off-topic. I'll remove this change from this PR and put it in another.

Reply:
https://github.com/brenner-tobias/addon-cloudflared/pull/51